### DISCUSSION TITLE
obsidian sync for android

### DISCUSSION THREAD

**Discussion by dashnaso** [st3v3nmw/obsidian-spaced-repetition#1168] [Q&A]
hi, has anyone got obsidian sync to import the csv file. mine does not sync the review progress for flashcards. i am using open obsidian spaced repetition recall with fsrs which uses a csv file for the review progress.

**Comment by KyleKlus** [comment]
Sry this is not the obsidian spaced repetition recall plugin (it is just the obsidian spaced repetition plugin, the other one is a separate fork), please ask over there if you have any questions regarding it 😊
